### PR TITLE
DDF-2824 fix for plugin diagrams and tables.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

- fixes incorrectly formatted diagrams.
- adds configuration info to Expiration Date plugin.

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 

#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)
verify content

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc [hunk 1]
```diff
@@ -8,4 +8,13 @@ The Expiration Date Pre-Ingest Plugin is installed by default with a standard in
 
 ===== Configuring the Expiration Date Pre-Ingest Plugin
 
-The Expiration Date Pre-Ingest Plugin has no configurable properties.
+To configure the Expiration Date Pre-Ingest Plugin:
+
+. Navigate to the ${admin-console}.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: ❗️ `${admin-console}` should be bold.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc [hunk 2]
```diff
@@ -8,4 +8,13 @@ The Expiration Date Pre-Ingest Plugin is installed by default with a standard in
...
+. Select the ${ddf-catalog} application.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: ❗️ `${ddf-catalog}` should be bold.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc [hunk 3]
```diff
@@ -8,4 +8,13 @@ The Expiration Date Pre-Ingest Plugin is installed by default with a standard in
...
+. Select the *Configuration* tab.
+. Select the Expiration Date Pre-Ingest Plugin.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: ❗️ Bold as well.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc [hunk 4]
```diff
@@ -8,4 +8,13 @@ The Expiration Date Pre-Ingest Plugin is installed by default with a standard in
...
+
+The expiration Date Pre-Ingest Plugin has these configurable properties:
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: ❗️ Capitalize 'expiration'

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_plugins/expiration-date-pre-ingest-plugin-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+|Boolean
+|If this is checked, overwrite all 'existing' non-empty expiration dates in metacards with a new date. If this is not checked, leave metacards with an existing expiration date.
+|false
+|true
+
+|Offset from Created Date (in days)
+|offsetFromCreatedDate
+|Integer
+|A metacard's new expiration date is calculated by adding this value (in days) to its created date.
+|true
```
Comment (lessarderic) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc]: ❗️ Shouldn't those be the other way around, i.e., `Default Value` = `30` and `Required` = `true`?

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Overwrite If Empty
+|overwriteIfBlank
+|Boolean
+|If this is checked, overwrite all 'blank' expiration dates in metacards. If this is not checked, leave metacards with 'blank' expiration dates as-is.
```
Comment (lessarderic) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc]: ❓ Any reason we single-quote _'blank'_ and _'existing'_ (below)?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc]: Not that I can tell. Removing.

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc (hunk 5 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.plugin.expiration.ExpirationDatePlugin-table-contents.adoc]: will fix.

**Pull request merged**